codechain/keystore/StorageJson.py

```diff
@@ -0,0 +1,50 @@
+import binascii
+import os
+import uuid
+
+from Crypto.Cipher import AES
+
+from ..crypto import blake256
+from ..crypto import get_public_from_private
+from .Errors import ErrorCode
+from .Errors import KeystoreError
+from .keys import key_from_public_key
+from .KeyType import KeyType
+from .Pbkdf2Async import pbkdf2_async
+from .Types import SecretStorage
+
+
+async def encode(private_key: str, key_type: KeyType, passphrase: str, meta: str):
+    public_key = get_public_from_private(private_key)
+    address = key_from_public_key(key_type, public_key)
+    salt = os.urandom(32)
+    iv = os.urandom(16)
+
+    kdf = "pbkdf2"
+    kdf_params = {
+        "dklen": 32,
+        "salt": binascii.hexlify(salt).decode("ascii"),
+        "c": 262144,
+        "prf": "hmac-sha256",
+    }
+    derived_key = await pbkdf2_async(
+        passphrase, salt, kdf_params.get("c"), kdf_params.get("dklen"), "sha256"
```

Comment:
Use the sync version instead of the async version.

Reply:
I changed it.